code/modules/mob/living/carbon/xenomorph/damage_procs.dm

```diff
@@ -121,7 +121,8 @@
 		"penetration" = penetration,
 		"armour_break_pr_pen" = armour_break_pr_pen,
 		"armour_break_flat" = armour_break_flat,
-		"armor_integrity" = armor_integrity
+		"armor_integrity" = armor_integrity,
+		"armour_type" = armour_type,
```

Comment:
should be armor not armour

Reply:
I'd normally agree but that'd be inconsistent with the other uses of the word "armour" in the damagedata lists throughout the code, and it's out of scope to fix that here